src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -12,6 +12,9 @@
 import org.testng.annotations.Test;
 
 import java.io.File;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
 
 public class JBWAIntegrationTest extends BaseTest {
```

Comment:
I didn't create name this file originally, and I'm not quite sure what the strict definition of integration test is for us. I think I'll leave it for now..